bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/util/OperationUtil.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.util;
+
+import com.google.api.client.util.BackOff;
+import com.google.api.client.util.ExponentialBackOff;
+import com.google.api.core.InternalApi;
+import com.google.common.primitives.Ints;
+import com.google.longrunning.GetOperationRequest;
+import com.google.longrunning.Operation;
+import com.google.longrunning.OperationsGrpc;
+import io.grpc.protobuf.StatusProto;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+@InternalApi("For internal usage only")
+public class OperationUtil {
+
+  @InternalApi("For internal usage only")
+  public static Operation getOperation(
+      GetOperationRequest request, OperationsGrpc.OperationsBlockingStub operationsStub) {
+    return operationsStub.getOperation(request);
+  }
+
+  @InternalApi("For internal usage only")
+  public static void waitForOperation(
+      Operation operation, OperationsGrpc.OperationsBlockingStub operationsStub)
+      throws IOException, TimeoutException {
+    waitForOperation(operation, 10, TimeUnit.MINUTES, operationsStub);
+  }
+
+  @InternalApi("For internal usage only")
+  public static void waitForOperation(
+      Operation operation,
+      long timeout,
+      TimeUnit timeUnit,
+      OperationsGrpc.OperationsBlockingStub operationsStub)
```

Comment:
Why make all of the methods static? You need the operationsStub for every method, why not keep the operationsStub as an instance variable and make all of the methods instance?